docs/editors/emacs.md

```diff
@@ -44,19 +51,18 @@ dependencies
 
 ;; Enable nice rendering of diagnostics like compile errors.
 (use-package flycheck
-  :ensure t
   :init (global-flycheck-mode))
-(use-package lsp-ui)
-(require 'lsp-ui)
 
-(use-package lsp-mode)
-(add-hook 'lsp-mode-hook 'lsp-ui-mode )
-(require 'lsp-mode)
-(require 'sbt-mode)
+(use-package lsp-mode
+  :pin melpa-stable)
+
+(use-package lsp-ui
+  :pin melpa-stable
+  :hook (lsp-mode . lsp-ui-mode))
 
-(add-to-list 'load-path "~/path/to/lsp-scala")
-(require 'lsp-scala)
-(add-hook 'scala-mode-hook #'lsp-scala-enable)
+(use-package 'lsp-scala
```

Comment:
I didn't catch this one until I copied and pasted: I don't think this should be quoted.

```suggestion
(use-package lsp-scala
```

Reply:
Thanks for that one! I missed that too!